hw/top_earlgrey/data/ip/chip_sysrst_ctrl_testplan.hjson

```diff
@@ -18,6 +18,10 @@
       si_stage: SV3
       lc_states: ["DEV", "PROD", "PROD_END", "RMA"]
       tests: ["chip_sw_pwrmgr_sysrst_ctrl_reset", "chip_sw_sysrst_ctrl_reset"]
+      bazel: [
+        "//sw/device/tests:pwrmgr_deep_sleep_all_reset_reqs_test",
```

Comment:
I noticed in pwrmgr_sleep_all_wake_ups_impl.c we just clear the wakeup_status CSR, but don't check it was set.

Reply:
I had a look at this but it seems that pwrmgr_sleep_all_wake_ups_impl.c has `check_wakeup_reason` which is called and that checks the wakeup reason? I am missing something?

Also I noticed that in the sival test, the code sets up a combo with all keys:
https://github.com/lowRISC/opentitan/blob/master/sw/device/tests/pwrmgr_sleep_resets_lib.c#L176
but actually only configures the pinmux for Key0. Should the combo be set up for Key0 only?